project.clj

```diff
@@ -7,16 +7,18 @@
                  [org.clojure/tools.cli "0.3.1"]
                  [cheshire "5.3.1"]
                  [clj-aws-s3 "0.3.10"]
-                 [clout "2.1.0"]
-                 [compojure "1.3.2"]
+                 [compojure "1.4.0"]
+                 [consul-clojure "0.1.0-SNAPSHOT"]
```

Comment:
Why would like to avoid it ?
This here is only library that abstracts all the querying for me. DNS based querying will also be consul specific.